.git-hooks/README.md

```diff
@@ -0,0 +1,31 @@
+# AWS SDK Code Examples - Git hooks
+
+This repository is configured with [git hooks](https://git-scm.com/docs/githooks). These hooks
+run automated code-quality checks like unit tests and linters.
+
+## Installation
+
+By default, git checks `.git/hooks` for existing hooks. This repository hosts hooks
+in a different location so they can be version controlled. Configure git to use these
+hooks by running `git config core.hooksPath .git-hooks`.
+
+## Usage
+
+A hook is triggered when its corresponding event takes place. For example,
+the "pre-commit" hook is triggered when a user initializes a commit, but before
+that commit is created in the git history.
+
+Hooks can be added to the `.git-hooks` folder. For a list of valid hooks, see https://git-scm.com/docs/githooks.
+
+## Existing hooks
+
+### [pre-commit](./pre-commit)
```

Comment:
Not sure how I feel about a title also being a hyperlink but I'll leave this one up to you.

Reply:
2-way door :)